src/BtTreeModel.cpp

```diff
@@ -480,7 +480,10 @@ bool BtTreeModel::insertRow(int row, const QModelIndex &parent, QObject* victim,
 
    bool success = true;
 
-   beginInsertRows(parent,row,row);
+   if(victimType == BtTreeItem::FOLDER)
+      beginInsertRows(QModelIndex(),row,row);
```

Comment:
`QModelIndex()` constructs an invalid index; [see here](http://doc.qt.io/qt-5/qmodelindex.html#QModelIndex). The usual way to create indices is [QAbstractItemModel::createIndex()](http://doc.qt.io/qt-5/qabstractitemmodel.html#createIndex).

@mikfire: any suggestion?